homeassistant/components/bluesound/media_player.py

```diff
@@ -691,7 +691,7 @@ def rebuild_bluesound_group(self) -> list[str]:
             reverse=True,
         )
         return [
-            cast(str, entity.name)
+            entity._sync_status.name  # noqa: SLF001 has to be done because this was accessible as entity.name before
```

Comment:
why not anymore? Can we make sync_status a public variable?

Reply:
On your request. I `_attr_name` is now set to `None`. I can make sync_status public.

Reply:
Oh right, that's true :)